engineering/core: Clarify weekly check-in schedule (#517)
sourcegraph/about: handbook/engineering/core-services/index.md

### Weekly check-ins

Everyone writes a weekly check-in in the active tracking issue of the on-going iteration. These check-ins describe the work done in the previous week as well as what's to come in the next week.

This process helps each teammate reflect and plan each week proactively and enables other parts of the organization to have visibility into on-going progress.

> Both Friday end of day and Monday beginning of the day are appropriate times to write this check-in.

Reviewer: I think this might be easier to understand:

```suggestion
Appropriate times to write this check-in are end of day on Friday or beginning of day on Monday.
```

But that comes from a mixture of gut feeling and personal taste. Feel free to ignore.

> Here's an example of a [good weekly check-in](https://github.com/sourcegraph/sourcegraph/issues/7190#issuecomment-573564817).

Reviewer: I think we should talk about why you picked @keegancsmith's and not mine in our next 1:1.

Author: Lol, it's just *one* good example (out of many others, including yours)!
